src/status_im/contexts/communities/sharing/events.cljs

```diff
@@ -70,6 +69,25 @@
                                   :chat-id chat-id
                                   :event   :communities/get-community-channel-share-data}))}]})))
 
+(rf/reg-event-fx :communities/share-community-url-with-data
+ (fn [_ [community-id]]
+   (let [title      (i18n/label :t/channel-on-status)
```

Comment:
This copy is for channels and not communities.

Reply:
Nice catch 🙌